api/envoy/extensions/io/io_uring/v3/io_uring.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
```

Comment:
Why isn't this in the bootstrap extension hierarchy? Looking below, I think this is what it is. This hierarchy is introduced in https://github.com/envoyproxy/envoy/pull/19467/files, it seems out bootstrap extensions have been adhoc scattered so far :(